src/game.cpp

```diff
@@ -5071,9 +5071,9 @@ void game::list_missions()
             }
         }
 
-        draw_tab(w_missions, 7, _("ACTIVE MISSIONS"), (tab == 0) ? true : false);
-        draw_tab(w_missions, 30, _("COMPLETED MISSIONS"), (tab == 1) ? true : false);
-        draw_tab(w_missions, 56, _("FAILED MISSIONS"), (tab == 2) ? true : false);
+        draw_tab( w_missions, 7, _( "ACTIVE MISSIONS" ), ( tab == 0 ) );
+        draw_tab( w_missions, 30, _( "COMPLETED MISSIONS" ), ( tab == 1 ) );
+        draw_tab( w_missions, 56, _( "FAILED MISSIONS" ), ( tab == 2 ) );
```

Comment:
Please, eliminate also the parentheses. They were a part of defensive programming (in case of adding yet another condition to the ternary operators having very low precedence), but add no value any more.